.github/workflows/go-tip-checks.yml

```diff
@@ -0,0 +1,119 @@
+name: Go-tip
+
+on:
+  schedule:
+    # Every Sunday at 13:00
+    - cron: "0 13 * * 0"
+
+jobs:
```

Comment:
I guess we can just trigger all other workflows in this repo, but with `tip` as input. It should be possible.

https://github.com/percona-platform/checked/blob/54e1cde7100f25dd3b5b1fd2d076b9c2bd69ffd2/.github/workflows/ci.yml#L343